code/game/machinery/vending.dm

```diff
@@ -1032,7 +1032,7 @@
 					/obj/item/weapon/tool/wrench = 5,/obj/item/device/analyzer = 5,/obj/item/device/t_scanner = 5,/obj/item/weapon/tool/screwdriver = 5,
 					/obj/item/device/flashlight/glowstick = 3, /obj/item/device/flashlight/glowstick/red = 3, /obj/item/device/flashlight/glowstick/blue = 3,
 					/obj/item/device/flashlight/glowstick/orange =3, /obj/item/device/flashlight/glowstick/yellow = 3)
-	contraband = list(/obj/item/weapon/weldingtool/hugetank = 2,/obj/item/clothing/gloves/fyellow = 2,)
+	contraband = list(/obj/item/weapon/weldingtool/hugetank = 2) //VORE: Removes budget insulated gloves.

```

Comment:
I would say leave this, since a vorestation edit would not be necessary. Sure ordinary yellow gloves in the hacked youtool would be wierd but eh, contraband lists have had strange, unrelated things before.